elements/pfe-accordion/src/pfe-accordion-header.scss

```diff
@@ -63,3 +64,38 @@ button[aria-expanded="true"] {
     @include pfe-accordion-variables($state: expanded, $theme: #{$context});
   }
 }
+
+/// ===========================================================================
+/// DISCLOSURE STYLES
+/// ===========================================================================
+
+:host([pfe-disclosure="true"]) {
+  button {
+    @include pfe-trigger($align: right);
+
+    &[aria-expanded="true"] {
+      --pfe-accordion--BorderLeftWidth: 4px;
```

Comment:
If you set the BorderLeftWidth on the `pfe-accordion`, it will propagate down to both the header and the panel.

Reply:
I'm going to try to set a new argument in `pfe-accordion-props` mixin to contain these styles.